README.md

```diff
@@ -1,12 +1,34 @@
-![CI](https://github.com/nearform/hub-template/actions/workflows/ci.yml/badge.svg?event=push)
+# node-test-github-summary
 
-# Hub Template
+![CI](https://github.com/nearform/node-test-github-summary/actions/workflows/ci.yml/badge.svg?event=push)
 
-A feature-packed template to start a new repository on the hub, including:
+A Github Summary test reporter for the Node.js test runner
 
-- code linting with [ESlint](https://eslint.org) and [prettier](https://prettier.io)
-- pre-commit code linting and commit message linting with [husky](https://www.npmjs.com/package/husky) and [commitlint](https://commitlint.js.org/)
-- dependabot setup with automatic merging thanks to ["merge dependabot" GitHub action](https://github.com/fastify/github-action-merge-dependabot)
-- notifications about commits waiting to be released thanks to ["notify release" GitHub action](https://github.com/nearform/github-action-notify-release)
-- PRs' linked issues check with ["check linked issues" GitHub action](https://github.com/nearform/github-action-check-linked-issues)
-- Continuous Integration GitHub workflow
+![Summary](docs/summary.png)
+
+## Installation
+
+```shell
+npm i -D node-test-github-summary
+```
+
+## Usage
+
+```shell
+node --test --test-reporter node-test-github-summary --test-reporter-destination $GITHUB_STEP_SUMMARY
```

Comment:
as discussed, let's make this the default so the user doesn't have to specify any destination when using this reporter